apis/securesourcemanager/v1alpha1/securesourcemanagerinstance_types.go

```diff
@@ -72,14 +35,36 @@ type SecureSourceManagerInstanceSpec struct {
 	// +optional
 	ResourceID *string `json:"resourceID,omitempty"`
 
+	/* NOTYET
 	// Optional. Labels as key value pairs.
 	Labels map[string]string `json:"labels,omitempty"`
+	*/
 
 	// Optional. Immutable. Customer-managed encryption key name, in the format
 	//  projects/*/locations/*/keyRings/*/cryptoKeys/*.
 	KmsKey *string `json:"kmsKey,omitempty"`
 }
 
+// SecureSourceManagerInstanceStatus defines the config connector machine state of SecureSourceManagerInstance
+type SecureSourceManagerInstanceStatus struct {
```

Comment:
I think we need some kind of common status here otherwise it may be unsafe to pull the `setStatus` to a helper func lib.  (related to [comment ](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2422/files#diff-24ab95ca167f1e4f6196bb2d4a04612afe7e3a22f4e9e7641d5b5c9a605bf3f9R332)